### DISCUSSION TITLE
REST API - where do we file possible bugs?

### DISCUSSION THREAD

**Discussion by fabiox23** [ant-media/Ant-Media-Server#5363] [General]
As in the title, in which section of the github repository do we file possible bugs in the REST API?

Thanks.

**Comment by yashtandon113** [comment]
@fabiox23 
I hope you are doing well.
You can report the bug on the Ant Media Repository. Here is the link to create the issue.
https://github.com/ant-media/Ant-Media-Server/issues/new/choose
